### DISCUSSION TITLE
New reaction type.. "unicorn"

### DISCUSSION THREAD

**Discussion by jayenne** [cybercog/laravel-love#172] [Ideas]
Hi,
would there be insterest in creating a new reaction type where a user may only have one unique (unicorn) for each model?
for example: I can love, fave, clap, subscribe  a lot of posts but I can only give one post my "unicorn" vote.
simply put, as i go about reading posts I can give one (and only one) post my "unicorn" to show the author tha this is my absolute favorite of any post. as i continue, fin find another post I like even more so i give that my uniucorn ...the app then simply reassigns my unicorn to this post instead.

*I'm assuning this can't be done by another name already?

**Comment by antonkomarev** [comment]
Hi @jayenne!
This feature looks like part of the #75 
It's like `MutuallyExclusiveReaction`, but even more exclusive :rofl:
